### PR TITLE
PMM-7337 STT check intervals

### PR DESCRIPTION
**Pull request by nicolalamacchia**
- [ ] Build: https://github.com/Percona-Lab/pmm-submodules/pull/1566
- [ ] https://github.com/percona/pmm-managed/pull/676
- [x] https://github.com/percona/pmm/pull/687

### CODE REVIEW

File: server/settings_test.go [hunk 1]
```diff
@@ -519,6 +525,51 @@ func TestSettings(t *testing.T) {
...
+							DefaultInterval: "0.9s",
+						},
+					},
+					Context: pmmapitests.Context,
+				})
+				pmmapitests.AssertAPIErrorf(t, err, 400, codes.InvalidArgument, `default_interval: minimal resolution is 1s`)
+				assert.Empty(t, res)
+			})
+
+			t.Run("HRFractional", func(t *testing.T) {
```
Comment (artemgavrilov) [inline server/settings_test.go:558]: Seems that name was copy-pased

**Review by artemgavrilov — APPROVED** [review]

**Comment by artemgavrilov** [comment]
Build failed, please run `dep ensure`

File: server/settings_test.go (hunk 1 above)
Comment (nicolalamacchia) [reply server/settings_test.go]: Oops, fixed

File: server/settings_test.go [hunk 2]
```diff
@@ -35,6 +35,12 @@ func TestSettings(t *testing.T) {
 			Lr: "60s",
 		}
 		assert.Equal(t, expected, res.Payload.Settings.MetricsResolutions)
+		expectedSTTCheckIntervals := &server.GetSettingsOKBodySettingsSttCheckIntervals{
+			FrequentInterval: "14400s",
+			DefaultInterval:  "86400s",
```
Comment (palash25) [inline server/settings_test.go:40]: should be named standard

Comment (nicolalamacchia) [reply server/settings_test.go]: Done

**Review by BupycHuk — APPROVED** [review]

File: server/settings_test.go [hunk 3]
```diff
@@ -655,6 +706,63 @@ func TestSettings(t *testing.T) {
 				})
 			})
 
+			t.Run("STTCheckIntervalsOK", func(t *testing.T) {
```
Comment (palash25) [inline server/settings_test.go:709]: `STTCheckIntervalsOK` -> `STTCheckIntervalsValid` since the previous test is named `..Invalid`

Comment (nicolalamacchia) [reply server/settings_test.go]: Done

**Review by palash25 — APPROVED** [review]

**Pull request merged**